multisense_ros/src/pps.cpp

```diff
@@ -124,12 +124,22 @@ void Pps::ppsCallback(const pps::Header& header)
 
 void Pps::connect()
 {
-    __sync_fetch_and_add(&subscribers_, 1);
+    const auto sub_count = __sync_fetch_and_add(&subscribers_, 1);
```

Comment:
Can we remove `auto` to make this compliant with pre-c++11 compilers.

Reply:
Sure, makes sense.